### DISCUSSION TITLE
Does PandaCSS work with server RSCs?

### DISCUSSION THREAD

**Discussion by kylemh** [chakra-ui/panda#1504] [Q&A] [answered]
[Next.js documentation outlines that PandaCSS works for client components in RSC; however, its not mentioned for an option with server components.](https://nextjs.org/docs/app/building-your-application/styling/css-in-js)

**Comment by astahmer** [comment, marked as answer]
yes, our own landing page https://panda-css.com/ is a Next App using RSC, you can see the source [here](https://github.com/chakra-ui/panda/tree/0f3bede5a785a125f94eb89c79e65ce0038c5977/website)
